test/tests.cpp

```diff
@@ -2352,3 +2352,17 @@ TEST_CASE_METHOD(QInterfaceTestFixture, "test_sqrtswap_reg")
     REQUIRE_FLOAT(qftReg->Prob(0), 0);
     REQUIRE_FLOAT(qftReg->Prob(1), 0);
 }
+
+TEST_CASE_METHOD(QInterfaceTestFixture, "test_qfusion_order")
```

Comment:
What's the coverage, now?  Does seem like you might want a few more test cases.

Reply:
See the PR that follows this one.